app/services/hotkeys.ts

```diff
@@ -504,46 +561,86 @@ export class Hotkey implements IHotkey {
     this.shouldApply = this.action.shouldApply(entityId);
   }
 
-
-  isSameHotkey(other: IHotkey) {
-    return (this.actionName === other.actionName) &&
-      (this.sceneId === other.sceneId) &&
-      (this.sourceId === other.sourceId) &&
-      (this.sceneItemId === other.sceneItemId);
+  equals(other: IHotkey) {
+    return (
+      this.actionName === other.actionName &&
+      this.sceneId === other.sceneId &&
+      this.sourceId === other.sourceId &&
+      this.sceneItemId === other.sceneItemId
+    );
   }
 
-
   getModel(): IHotkey {
     return { ...this.hotkeyModel };
   }
 
-
   private getAction(entityId: string): IHotkeyAction {
-    const action = { ...HOTKEY_ACTIONS[this.type].find(action => {
-      return action.name === this.actionName;
-    }) };
+    const action = getActionFromName(this.actionName);
 
     const { up, down } = action;
 
     // Fill in optional functions with some defaults
-    if (!action.isActive) action.isActive = () => false;
-    if (!action.shouldApply) action.shouldApply = () => true;
-
+    if (!action.isActive) {
+      action.isActive = () => false;
+    }
+    if (!action.shouldApply) {
+      action.shouldApply = () => true;
+    }
 
     // We defer the actions until after we've decided whether
     // or not to execute each action.
     if (up) {
       action.upHandler = () => {
-        if (!action.isActive(entityId)) defer(() => up(entityId));
+        if (!action.isActive(entityId)) {
+          defer(() => up(entityId, this.hotkeyModel.hotkeyId));
+        }
       };
     }
 
     if (down) {
       action.downHandler = () => {
-        if (!action.isActive(entityId)) defer(() => down(entityId));
+        if (!action.isActive(entityId)) {
+          defer(() => down(entityId, this.hotkeyModel.hotkeyId));
+        }
       };
     }
 
     return action;
   }
 }
+
+const getMigrationMapping = (actionName: string) => {
+  return {
+    MUTE: 'TOGGLE_MUTE',
+    UNMUTE: 'TOGGLE_UNMUTE',
+  }[normalizeActionName(actionName)];
+};
+
+const getActionFromName = (actionName: string) => {
+  return ACTIONS[actionName] || ACTIONS[getMigrationMapping(actionName)];
+};
```

Comment:
not sure why this and the above are using manual returns

Reply:
I've changed `getActionFromName`. I believe `getMigrationMapping` should stay this way due to a somewhat complex expression and the fact we're returning an object.